Classes/Job/Workspace/Publish.php

```diff
@@ -143,6 +142,7 @@ public function execute(QueueInterface $queue, Message $message): bool
         );
         $this->feedbackCollection->setControllerContext($controllerContext);
 
+        $job = $this->jobRepository->findByIdentifier($this->jobIdentifier);
```

Comment:
Versteh ich das richtig: Das läd den Job jetzt zweimal, aber immerhin haben wir dann in Zeile 145 einen Job den die Unit of Work kennt, auch falls sie erst mal neu verbindet muss dafür?

Reply:
Teilweise. Nachdem der erste job nicht gespeichert wird, sondern das if nur da ist um sicherzustellen, dass es überhaupt einen Job mit dieser ID gibt fände ich "re-load" nicht richtig.